coda/coda_mdstore/tests/test_views.py

```diff
@@ -591,22 +591,35 @@ def test_raises_http404_file_handle_is_falsy(self, rf):
         with pytest.raises(http.Http404):
             views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_output_text(self):
-        assert 0
+    def test_response_content(self, rf):
+        """Test the output contains domain and parsed tag attributes from getFileHandle."""
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
+        # Mock what gets read from the manifest file.
+        self.getFileHandle.return_value.readline.side_effect = [
+            '<html coda/bag/123/manifest-md5.txt', '<html coda/bag/123/bagit.txt', '']
+        bag = FullBagFactory.create()
+        request = rf.get('/')
+        response = views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_top_files_block(self):
-        # TODO: rename once implemented.
-        assert 0
+        assert b'https://coda/coda/bag/123/bagit.txt\nhttps://coda/coda/bag/123/manifest-md5.txt'\
+               in response.content
+        assert response.status_code == 200
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_path_is_not_unicode_safe(self):
-        assert 0
+    def test_response_content_has_topFiles(self, rf):
+        """Test topFiles are returned in the response."""
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
+        # Mock what gets read from the manifest file.
+        self.getFileHandle.return_value.readline.side_effect = [
+            '<html coda/bag/123/manifest-md5.txt', '<html coda/bag/123/bagit.txt', '']
```

Comment:
Again, for easier reading of the test,  I would change the `side_effect` values to be like mentioned above.

Reply:
this one too?